deployment/console-plugin-nvidia-gpu/templates/consoleplugin.yaml

```diff
@@ -1,13 +1,16 @@
-apiVersion: console.openshift.io/v1alpha1
+apiVersion: console.openshift.io/v1
 kind: ConsolePlugin
 metadata:
   name: {{ include "console-plugin-nvidia-gpu.fullname" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
FYI ConsolePlugin is a cluster-scoped resource so specifying namespace does nothing. Not a big deal though